polaris/polaris/migrations/0004_auto_20201120_1541.py

```diff
@@ -0,0 +1,23 @@
+# Generated by Django 2.2.16 on 2020-11-20 15:41
```

Comment:
This migration file is out of date compared to the changes I see. There is no update of the `status` column but you've added a new value to it. 

Since that value will be removed from `status` this looks good now but always make sure your migrations match the current definition of the model.

Reply:
Removed the `pending_anchor_claimable_start` status.